diff -ruN a/qtmultimedia/cmake/FindWMF.cmake b/qtmultimedia/cmake/FindWMF.cmake
--- a/qtmultimedia/cmake/FindWMF.cmake	2022-12-12 19:36:40.000000000 +0000
+++ b/qtmultimedia/cmake/FindWMF.cmake	2023-01-06 14:33:41.797015404 +0000
@@ -22,11 +22,11 @@
 find_library(WMF_MSDMO_LIBRARY msdmo HINTS ${CMAKE_C_IMPLICIT_LINK_DIRECTORIES})
 find_library(WMF_OLE32_LIBRARY ole32 HINTS ${CMAKE_C_IMPLICIT_LINK_DIRECTORIES})
 find_library(WMF_OLEAUT32_LIBRARY oleaut32 HINTS ${CMAKE_C_IMPLICIT_LINK_DIRECTORIES})
-find_library(WMF_MF_LIBRARY Mf HINTS ${CMAKE_C_IMPLICIT_LINK_DIRECTORIES})
-find_library(WMF_MFUUID_LIBRARY Mfuuid HINTS ${CMAKE_C_IMPLICIT_LINK_DIRECTORIES})
-find_library(WMF_MFPLAT_LIBRARY Mfplat HINTS ${CMAKE_C_IMPLICIT_LINK_DIRECTORIES})
-find_library(WMF_MFCORE_LIBRARY Mfcore HINTS ${CMAKE_C_IMPLICIT_LINK_DIRECTORIES})
-find_library(WMF_PROPSYS_LIBRARY Propsys HINTS ${CMAKE_C_IMPLICIT_LINK_DIRECTORIES})
+find_library(WMF_MF_LIBRARY mf HINTS ${CMAKE_C_IMPLICIT_LINK_DIRECTORIES})
+find_library(WMF_MFUUID_LIBRARY mfuuid HINTS ${CMAKE_C_IMPLICIT_LINK_DIRECTORIES})
+find_library(WMF_MFPLAT_LIBRARY mfplat HINTS ${CMAKE_C_IMPLICIT_LINK_DIRECTORIES})
+find_library(WMF_MFCORE_LIBRARY mfcore HINTS ${CMAKE_C_IMPLICIT_LINK_DIRECTORIES})
+find_library(WMF_PROPSYS_LIBRARY propsys HINTS ${CMAKE_C_IMPLICIT_LINK_DIRECTORIES})
 
 
 set(WMF_LIBRARIES ${WMF_STRMIIDS_LIBRARY} ${WMF_AMSTRMID_LIBRARY} ${WMF_DMOGUIDS_LIBRARY} ${WMF_UUID_LIBRARY}
diff -ruN a/qtmultimedia/src/multimedia/windows/qwindowsmediadevices.cpp b/qtmultimedia/src/multimedia/windows/qwindowsmediadevices.cpp
--- a/qtmultimedia/src/multimedia/windows/qwindowsmediadevices.cpp	2022-12-12 19:36:40.000000000 +0000
+++ b/qtmultimedia/src/multimedia/windows/qwindowsmediadevices.cpp	2023-01-06 15:16:47.561588092 +0000
@@ -13,7 +13,7 @@
 #include <mmddk.h>
 #include <mfobjects.h>
 #include <mfidl.h>
-#include <Mferror.h>
+#include <mferror.h>
 #include <mmdeviceapi.h>
 #include <qwindowsmfdefs_p.h>
 
diff -ruN a/qtmultimedia/src/multimedia/windows/qwindowsresampler.cpp b/qtmultimedia/src/multimedia/windows/qwindowsresampler.cpp
--- a/qtmultimedia/src/multimedia/windows/qwindowsresampler.cpp	2022-12-12 19:36:40.000000000 +0000
+++ b/qtmultimedia/src/multimedia/windows/qwindowsresampler.cpp	2023-01-06 15:15:54.221425824 +0000
@@ -7,7 +7,7 @@
 #include <qloggingcategory.h>
 #include <QUuid>
 
-#include <Wmcodecdsp.h>
+#include <wmcodecdsp.h>
 #include <mftransform.h>
 #include <mferror.h>
 
diff -ruN a/qtmultimedia/src/plugins/multimedia/windows/common/mfmetadata_p.h b/qtmultimedia/src/plugins/multimedia/windows/common/mfmetadata_p.h
--- a/qtmultimedia/src/plugins/multimedia/windows/common/mfmetadata_p.h	2022-12-12 19:36:40.000000000 +0000
+++ b/qtmultimedia/src/plugins/multimedia/windows/common/mfmetadata_p.h	2023-01-06 15:20:13.530268737 +0000
@@ -16,7 +16,7 @@
 //
 
 #include <qmediametadata.h>
-#include "Mfidl.h"
+#include "mfidl.h"
 
 QT_USE_NAMESPACE
 
diff -ruN a/qtmultimedia/src/plugins/multimedia/windows/decoder/mfaudiodecodercontrol.cpp b/qtmultimedia/src/plugins/multimedia/windows/decoder/mfaudiodecodercontrol.cpp
--- a/qtmultimedia/src/plugins/multimedia/windows/decoder/mfaudiodecodercontrol.cpp	2022-12-12 19:36:40.000000000 +0000
+++ b/qtmultimedia/src/plugins/multimedia/windows/decoder/mfaudiodecodercontrol.cpp	2023-01-06 15:21:08.886463597 +0000
@@ -4,7 +4,7 @@
 #include <system_error>
 #include <mferror.h>
 #include <qglobal.h>
-#include "Wmcodecdsp.h"
+#include "wmcodecdsp.h"
 #include "mfaudiodecodercontrol_p.h"
 #include <private/qwindowsaudioutils_p.h>
 
diff -ruN a/qtmultimedia/src/plugins/multimedia/windows/mediacapture/qwindowsmediadevicereader_p.h b/qtmultimedia/src/plugins/multimedia/windows/mediacapture/qwindowsmediadevicereader_p.h
--- a/qtmultimedia/src/plugins/multimedia/windows/mediacapture/qwindowsmediadevicereader_p.h	2022-12-12 19:36:40.000000000 +0000
+++ b/qtmultimedia/src/plugins/multimedia/windows/mediacapture/qwindowsmediadevicereader_p.h	2023-01-06 15:22:15.074701912 +0000
@@ -17,8 +17,8 @@
 
 #include <mfapi.h>
 #include <mfidl.h>
-#include <Mferror.h>
-#include <Mfreadwrite.h>
+#include <mferror.h>
+#include <mfreadwrite.h>
 
 #include <QtCore/qobject.h>
 #include <QtCore/qmutex.h>
diff -ruN a/qtmultimedia/src/plugins/multimedia/windows/mediacapture/qwindowsmediaencoder.cpp b/qtmultimedia/src/plugins/multimedia/windows/mediacapture/qwindowsmediaencoder.cpp
--- a/qtmultimedia/src/plugins/multimedia/windows/mediacapture/qwindowsmediaencoder.cpp	2022-12-12 19:36:40.000000000 +0000
+++ b/qtmultimedia/src/plugins/multimedia/windows/mediacapture/qwindowsmediaencoder.cpp	2023-01-06 15:25:12.227471654 +0000
@@ -8,7 +8,7 @@
 #include "mfmetadata_p.h"
 #include <QtCore/QUrl>
 #include <QtCore/QMimeType>
-#include <Mferror.h>
+#include <mferror.h>
 #include <shobjidl.h>
 #include <private/qmediastoragelocation_p.h>
 #include <private/qmediarecorder_p.h>
diff -ruN a/qtmultimedia/src/plugins/multimedia/windows/player/mfplayercontrol_p.h b/qtmultimedia/src/plugins/multimedia/windows/player/mfplayercontrol_p.h
--- a/qtmultimedia/src/plugins/multimedia/windows/player/mfplayercontrol_p.h	2022-12-12 19:36:40.000000000 +0000
+++ b/qtmultimedia/src/plugins/multimedia/windows/player/mfplayercontrol_p.h	2023-01-06 15:23:28.223564762 +0000
@@ -15,9 +15,9 @@
 // We mean it.
 //
 
-#include "QUrl.h"
 #include "private/qplatformmediaplayer_p.h"
 
+#include <QtCore/qurl.h>
 #include <QtCore/qcoreevent.h>
 
 QT_USE_NAMESPACE
diff -ruN a/qtmultimedia/src/plugins/multimedia/windows/player/mfplayersession.cpp b/qtmultimedia/src/plugins/multimedia/windows/player/mfplayersession.cpp
--- a/qtmultimedia/src/plugins/multimedia/windows/player/mfplayersession.cpp	2022-12-12 19:36:40.000000000 +0000
+++ b/qtmultimedia/src/plugins/multimedia/windows/player/mfplayersession.cpp	2023-01-06 15:24:30.617014400 +0000
@@ -32,7 +32,7 @@
 
 #include <mmdeviceapi.h>
 #include <propvarutil.h>
-#include <Functiondiscoverykeys_devpkey.h>
+#include <functiondiscoverykeys_devpkey.h>
 
 //#define DEBUG_MEDIAFOUNDATION
 
diff -ruN a/qtmultimedia/src/plugins/multimedia/windows/player/mftvideo.cpp b/qtmultimedia/src/plugins/multimedia/windows/player/mftvideo.cpp
--- a/qtmultimedia/src/plugins/multimedia/windows/player/mftvideo.cpp	2022-12-12 19:36:40.000000000 +0000
+++ b/qtmultimedia/src/plugins/multimedia/windows/player/mftvideo.cpp	2023-01-06 15:24:48.744327572 +0000
@@ -7,7 +7,7 @@
 #include <mferror.h>
 #include <strmif.h>
 #include <uuids.h>
-#include <InitGuid.h>
+#include <initguid.h>
 #include <d3d9.h>
 #include <qdebug.h>
 
diff -ruN a/qtmultimedia/src/plugins/multimedia/windows/qwindowsvideodevices.cpp b/qtmultimedia/src/plugins/multimedia/windows/qwindowsvideodevices.cpp
--- a/qtmultimedia/src/plugins/multimedia/windows/qwindowsvideodevices.cpp	2022-12-12 19:36:40.000000000 +0000
+++ b/qtmultimedia/src/plugins/multimedia/windows/qwindowsvideodevices.cpp	2023-01-06 15:31:51.473382127 +0000
@@ -8,11 +8,11 @@
 #include <private/qwindowsmultimediautils_p.h>
 #include <private/qwindowsiupointer_p.h>
 
-#include <Dbt.h>
+#include <dbt.h>
 
 #include <mfapi.h>
 #include <mfreadwrite.h>
-#include <Mferror.h>
+#include <mferror.h>
 
 QT_BEGIN_NAMESPACE
 
